src/components/app.cjsx

```diff
@@ -16,12 +19,10 @@ module.exports = React.createClass
     HistoryLocation.removeChangeListener(@storeHistory)
 
   storeInitial: ->
-    if History.length is 1
-      path = HistoryLocation.getCurrentPath()
-      TransitionActions.load({path})
+    @storeHistory(path: HistoryLocation.getCurrentPath())
 
```

Comment:
Nah, we don't need to. `router.transistionTo` can navigate with using complete path string that it previously had.